.github/workflows/smoke-test.yaml

```diff
@@ -40,24 +41,32 @@ jobs:
           restore-keys: |
             yarn-
 
-      - name: Install dependencies and Build Framework
+      - name: Install framework dependencies
         run: |
           yarn install --immutable
-          yarn build:clean || echo "Project already clean"
-          yarn build:js
 
       - name: Setup test project
         id: setup_test_project
         uses: ./.github/actions/setup_test_project
 
-      - name: Run `rw info`
+      - name: Install Playwright deps
+        run: npx playwright install --with-deps chromium
+
+      - name: Run `rw build`
         run: |
-          yarn rw info
+          yarn rw build
         working-directory: ${{ steps.setup_test_project.outputs.test_project_path }}
 
-      - name: Run `rw build` without prerender
+      # We run prisma reset, seed, rw dev, rw build and rw serve
+      - name: Run smoke tests
+        working-directory: ./tasks/smoke-test
+        run: npx playwright test
+        env:
+          PROJECT_PATH: ${{ steps.setup_test_project.outputs.test_project_path }}
```

Comment:
I don't think we need Telemetry data from these smoke tests, but it looks like only `rw dev` and `rw serve` are being run (no Telemetry for either). If I'm mistaken, lets add REDWOOD_DISABLE_TELEMETRY=1

Reply:
Done